functions/zoom-downloader.py

```diff
@@ -254,26 +252,35 @@ def _series_id_from_schedule(self):
             ("S", "Saturday"),
             ("U", "Sunday")
         ])
-        day_code = list(days.keys())[zoom_time.weekday()]
-        if day_code not in schedule["Days"]:
-            logger.debug("No opencast recording scheduled on {}."
-                         .format(days[day_code]))
-            return None
 
-        for t in schedule["Time"]:
-            scheduled_time = datetime.strptime(t, "%H:%M")
+        zoom_time = self._created_local
+        logger.info({"meeting creation time": zoom_time,
+                     "course schedule": schedule})
+        zoom_day_code = list(days.keys())[zoom_time.weekday()]
+
+        # events is a list of {title, day, time} dictionaries
+        for event in schedule["events"]:
+            # match day
+            if zoom_day_code != event["day"]:
+                continue
```

Comment:
Like airbnb eslint would complain about this `continue` here. I guess it wants you to just endessly nest `if` statements? IDK it really bugs me how wrong that is.